composer.json

```diff
@@ -37,10 +33,10 @@
       "WooCommerce\\GitHooks\\Hooks::postHooks"
     ],
     "phpcs": [
-      "phpcs -s -p"
+      "vendor/bin/phpcs"
```

Comment:
Inside `composer.json` you don't really need to set the full path.

Reply:
Better yet: set it to the actual script and add `@php` in front of it for optimal cross-platform compatibility. Your remark makes a few too many assumption about the platform devs use.